src/components/existing-request/nr-approved-gray-box.vue

```diff
@@ -82,3 +131,24 @@ export default class NrApprovedGrayBox extends Vue {
   readonly disabled: boolean
 }
 </script>
+
+<style lang="scss" scoped>
+@import '@/assets/styles/theme.scss';
+
+.v-btn {
+  margin-top: 30px;
+}
+
+.contact-registries {
+  font-size: $px-16;
+  margin-top: 30px;
+  .v-icon.v-icon {
+    color: $app-dk-blue;
+  }
+}
+
+.important-note {
+  margin-top: 30px;
+  margin-bottom: 30px;
+}
```

Comment:
Since we have multiple designs that spec out 30px whitespace, let's make this generic and add it to layout.scss.

I recommend 2 new classes: `mt-30` and `mb-30` that you can use on line 85 instead.

Reply:
Awesome, thanks!